storage/integration_test.go

```diff
@@ -1662,6 +1662,9 @@ func TestIntegration_Compose(t *testing.T) {
 		if _, err := c.Run(ctx); err != nil {
 			t.Fatalf("ComposeFrom error: %v", err)
 		}
+		if c.ComponentCount != int64(len(objects)) {
+			t.Fatalf("ComponentCount of dst Object: %v is incorrect", c.ComponentCount)
```

Comment:
This should be `Errorf` rather than `Fatalf` since it does not block continuing the test. Same for the other error below.

Also, you should specify "got" and "want" values in the errors.

Reply:
fixed